seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -21,6 +21,15 @@
 class TerraApiUtilsCase(TestCase):
     fixtures = ['users', 'social_auth']
 
+    def test_is_google_authenticated(self, mock_logger):
+        user = User.objects.get(email = 'test_user@test.com')
+        r = is_google_authenticated(user)
+        self.assertTrue(r)
+        user = User.objects.get(email = 'test_local_user@test.com')
+        r = is_google_authenticated(user)
+        self.assertFalse(r)
+        self.assertEqual(len(mock_logger.method_calls), 0)
```

Comment:
why test that nothing was logged? If there is a case where something should be logged, why not test it?

Reply:
The `mock_logger` is not been used in this test. But since it is a class modifier, I have to add it to the parameter of this test. Adding an assertion can avoid a warning of the parameter not used.

Reply:
but why do you need to mock the logger at all then?